packages/components/scripts/templates/__tests__/component.ct.tsx

```diff
@@ -0,0 +1,23 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { toKebabCase } from '../../transforms';
+import { type ComponentFileBuilderResponse } from '../../types';
+
+export const Setup = (componentName: string): ComponentFileBuilderResponse => ({
+    content: `/* (c) Copyright Frontify Ltd., all rights reserved. */
+import { expect, test } from '@playwright/experimental-ct-react';
```

Comment:
```suggestion
    content: `/* (c) Copyright Frontify Ltd., all rights reserved. */

import { expect, test } from '@playwright/experimental-ct-react';
```
Else eslint complains on the created file